app/Livewire/Auth/ForgotPassword.php

```
@@ -20,8 +24,60 @@ public function sendPasswordResetLink(): void
+    /**
+     * Rate limit the password reset process.
+     */
+    protected function throttle($attempts, $decay, $key): void
+    {
+        $this->ensureIsNotRateLimited($attempts, $decay, $key);
+    }
+
+    /**
+     * Ensure the authentication request is not rate limited.
+     */
+    protected function ensureIsNotRateLimited($attempts, $decay, $key): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$decay'. (Unused Code Rules)

(UnusedFormalParameter)